### PR TITLE
add bridge for control module

### PR DESCRIPTION
**Pull request by muleisheng**

### CODE REVIEW

File: modules/bridge/bridge_control_component.cc [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    FLAGS_chassis_pending_queue_size);
+  CreateReader(trajectory_reader_, FLAGS_planning_trajectory_topic,
+    FLAGS_planning_pending_queue_size);
+  CreateReader(localization_reader_, FLAGS_localization_topic,
+    FLAGS_localization_pending_queue_size);
+  CreateReader(pad_msg_reader_, FLAGS_pad_topic,
+    FLAGS_pad_msg_pending_queue_size);
+
+  AINFO << "Control resetting vehicle state, sleeping for 1000 ms ...";
+  std::this_thread::sleep_for(std::chrono::milliseconds(1000));
```
Comment (Capri2014) [inline modules/bridge/bridge_control_component.cc]: Double check if this is needed.

**Comment by Capri2014** [comment]
remember to later add a `./scripts/bridge_control.sh` for easier test

File: modules/bridge/BUILD [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        "//modules/common",
+        "//modules/common/adapters:adapter_gflags",
+        "//modules/common/monitor_log",
+        "//modules/common/time",
+        "//modules/common/util",
+        "//modules/control/common",
+        "//modules/control/controller",
+        "//modules/control/proto:control_proto",
+        "//modules/localization/proto:localization_proto",
+        "//modules/perception/proto:perception_proto",
```
Comment (Capri2014) [inline modules/bridge/BUILD]: why this depends on `perception_proto`?

File: modules/bridge/BUILD [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    deps = [
+        "//cyber",
+        "//external:gflags",
+        "//modules/canbus/proto:canbus_proto",
+        "//modules/common",
+        "//modules/common/adapters:adapter_gflags",
+        "//modules/common/monitor_log",
+        "//modules/common/time",
+        "//modules/common/util",
+        "//modules/control/common",
```
Comment (Capri2014) [inline modules/bridge/BUILD]: why this depends on sub-folder under control? It should only be depending on `control_proto` right?

File: modules/bridge/BUILD [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        "//modules/control/controller",
+        "//modules/control/proto:control_proto",
+        "//modules/localization/proto:localization_proto",
+        "//modules/perception/proto:perception_proto",
+        "//modules/planning/proto:planning_proto",
+        "//modules/prediction/proto:prediction_proto",
```
Comment (Capri2014) [inline modules/bridge/BUILD]: similarly for `prediction_proto`, remove redundant dependency please.

File: modules/bridge/BUILD [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+)
+
+cc_binary(
+    name = "libbridge_control_component.so",
+    linkshared = True,
+    linkstatic = False,
+    deps = [":bridge_control_component_lib"],
+)
+
+#cc_test(
```
Comment (Capri2014) [inline modules/bridge/BUILD]: Please re-enable the test and make sure at least I/O is right.

File: modules/bridge/bridge_control_component.cc [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+ * limitations under the License.
+ *****************************************************************************/
+#include "modules/bridge/bridge_control_component.h"
+
+#include "cyber/common/file.h"
+#include "cyber/common/log.h"
+#include "modules/common/adapters/adapter_gflags.h"
+#include "modules/common/time/time.h"
+#include "modules/common/vehicle_state/vehicle_state_provider.h"
+#include "modules/control/common/control_gflags.h"
```
Comment (Capri2014) [inline modules/bridge/bridge_control_component.cc]: why do we need to include `control_gflags` here?

**Comment by Capri2014** [comment]
Also the commit message format should be following: "ModuleName: commit_content"

File: modules/bridge/bridge_control_component.h [hunk 7]
```diff
@@ -0,0 +1,90 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
+ *
```
Comment (jinghaomiao) [inline modules/bridge/bridge_control_component.h]: 2019

File: modules/bridge/dag/bridge_control.dag [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+    module_library : "/apollo/bazel-bin/modules/bridge/libbridge_control_component.so"
+    timer_components {
+        class_name : "BridgeControlComponent"
+        config {
+            name: "control"
+            flag_file_path: "/apollo/modules/bridge/conf/control.conf"
+            interval: 10
+      }
+   }
+}
```
Comment (jinghaomiao) [inline modules/bridge/dag/bridge_control.dag]: 我理解我们需要两个dag，一个负责UDP分发planning，是通过planning数据驱动的，一个负责收集chassis feedback，是timer驱动的。

**Pull request closed without merging**